### PR TITLE
Port over `pkgdown::in_pkgdown()`

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
For #1739

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -41,7 +41,6 @@ Suggests:
     jsonlite,
     mockery,
     patrick,
-    pkgdown,
```
Comment (AshesITR) [inline DESCRIPTION:44]: Is putting pkgdown in Suggests not the usual way of signaling we use it for development?

Comment (IndrajeetPatil) [reply DESCRIPTION:44]: Nope.

Comment (IndrajeetPatil) [reply DESCRIPTION:44]: You can sample any of the r-lib, tidyverse, etc. organizations packages, all of which use _pkgdown_, but none of them have _pkgdown_ in `Suggests`. As far as CRAN is concerned, it's irrelevant.

Comment (AshesITR) [reply DESCRIPTION:44]: Okay, thanks for clarifying.

**Review by AshesITR — APPROVED** [review]

**Pull request merged**